src/priority_fee.rs

```diff
@@ -279,7 +278,7 @@ impl PriorityFeeTracker {
         &self,
         accounts: Vec<Pubkey>,
         include_vote: bool,
-        lookback_period: Option<usize>,
+        lookback_period: Option<u32>,
```

Comment:
No functional reason to change from usize to u32 but I feel that people will have no clue what is usize in error. maybe u32 will be a little more descriptive. Not a good reason to change type but I could not find any good way to adjust the error message in serde